blog/crossplane-nvim.md

```
@@ -0,0 +1,55 @@
+title: "Crossplane resources in Neovim"
+date: 2024-03-31
+tags: ["crossplane", "neovim"]
+# Introduction
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)